### DISCUSSION TITLE
Zwei Erweiterungen

### DISCUSSION THREAD

**Discussion by Pepe844** [RolandRust/HaniMandlWroom#11] [Ideas]
Hallo, ich bin ein großer Fan dieses Projektes und als Imker mit 10 Völkern ist diese Lösung schon ein mächtiges Tool. Folgende Ideen zur Erweiterung des funktionalen Umfangs hätte ich in der Tat:
1. Ich habe desöfteren die Herausforderung, dass der **Honig unterschiedliche Konsistenzen** (Cremig vs. Flüssig) hat, was dazu führt, dass man manuell den Öffnungswinkel nachstellen muss, da sonst bei hoher Fließgeschwindigkeit zu viel Honig im Glas landet. Folgenden Vorschlag dazu: Menüerweiterung -> Konsitenz >Auswahl: Flüssig / cremig (Einstellung führt zur Voreinstellung des Öffnungswinkels)

2. Eine weitere Herausforderung ist das Abfüllen in **unterschiedlichen 500g Gläser**n. Diese landen in unterschiedlichen Stückzahlen bei mir und werden nach entsprechender Reinigung wiederverwendet. Aktuell gehe ich dafür in das TARA Menü und kalibriere das Glas für die Abfüllung neu. Ich liebe den Automatischen Modus, da dieser ein flüssiges Arbeiten ermöglicht. Und genau hier wäre es hilfreich, wenn das System mit dem Aufstellen des neuen 500g - Glases eine Nullkalibrierung durchführt und dann 500g Honig abfüllt. 

Frohes Fest und einen guten Rutsch!

**Comment by RolandRust** [comment]
@Pepe844 
1: Kann man bei Gelegenheit machen.  So nur mit einer Menüerweiterung ist das nicht getan. Es bräuchte dann auch zwei Werte für  den Öffnungswinkel. Meiner Ansicht nach  sollte dann auch der Feindossierungswinkel individuell eingestellt werden können.  Könnte man im Servomenü integrieren, da ja man in diesem Menü die Winkel festlegt.

2: Der HM macht das eigendlich schon. Wenn ein neues Glass draufgestellt wird im Automatik Modus, wird eine Nullkalibrierung durchgeführt. Es gibt da aber eine Bedingung die erfüllt werden muss. Die Abweichung von dem Gewicht zum dem gespeicherten Tarre Wert im HM für diesen Glastyp darf nicht mehr Abweichen als die +- Glasstolleranz, welche im Menü Automatik eingestellt wurde.  Diese kann von 0g bis 99g eingestellt werden. Denke auch wenn du unterschiedliche Glasssorten verwendest, werden die nicht 198g unterschied haben.
Es ist aber zu beachten, jeh höher die Glasstolleranz eingestellt ist, desto wahrscheinlicher ist es, das der HM ein kleineres oder Grösseres Glass als richtig annimmt und der Abfüllvorgang startet.

Zu deinem Problem im Punkt 2: Versuche mal ein mittelschweres Glass  als Tare Vorlage für den HM und erhöhe die Glasstolleranz.  Denke so wirst du deine unterschiedlichen Glässer ohne ein neues Tate zu machen füllen können.

Gruss Roli
